README.md

```diff
@@ -22,15 +22,19 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 ## Screenshots
 
-![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot1.png "Main page")
+![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/ss_main_page.png "Main page")
```

Comment:
There's a decent amount of blank space in the lower half of this image - possibly resize the browser window to eliminate it and redo the screenshot?

Reply:
Done